content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,187 @@
+    = P_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+
+The transition of the Up and Down component inside a layer can then be write as such
+
+ .. figure:: images/InsideLayer.png
+    :align: center
+    :scale: 100% 
+    :name: InsideLayer
+
+    Transition inside a layer, variables definition.
+
+
+.. math::
+    \left(\begin{matrix} U_j' \\ D_j' \end{matrix} \right)  = \left(\begin{matrix} e^{i k h_j} & 0 \\ 0 & e^{-i k h_j} \end{matrix} \right) \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+    = T_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+
+With the variables U, D, U' and D' define as in (:numref:`InsideLayer`)
+
+Using the continuity of the tangential \\(\\mathbf{E_x}\\) and \\(\\mathbf{H_y}\\) field at the interfaces, we find an iterative relation between the fields in consecutive layers:
+
+.. math::
+    \left(\begin{matrix} E_{x,j} \\ H_{y,j} \end{matrix} \right) = P_j T_j P^{-1}_J \left(\begin{matrix} E_{x,j+1} \\ H_{y,j+1} \end{matrix} \right)
+
+We are now only missing a Boundary Condition to be able to compute our MT forward modeling. A reasonable one is to set the Down Amplitude to 1 and the Up Amplitude to 0 in the last layer, as there is no reflection from an other interface below.
+
+.. math::
+    \left(\begin{matrix} U_n \\ D_n \end{matrix} \right)  = \left(\begin{matrix} 0 \\ 1 \end{matrix} \right) 
+
+
+Field Acquisition
+-----------------
+
+In MT, the source is unknown but we are avoiding the problem by measuring the ratio of the fields. usually at the surface. We define an apparent impedance
+
+.. math::
+    \hat{Z_{xy}} = \frac{E_x}{H_y}
+
+Notice this is a complex number, with a norm and an angle.
+
+Data
+----
+
+Apparent Resistivity
+********************
+
+.. math::
+    \rho_{app} = \frac{1}{\mu_0 \omega} |\hat{Z_{xy}}|
+
+For a half-space, \\(\\rho_{app} = \\rho_{earth}\\).
+
+For a unhomogeneous earth, \\(\\rho_{app}\\) at a particular frequency is an average of the conductivity of the earth on about a sphere with a radius equal to the skin depth.
+
+Phase
+*****
+
+.. math::
+    \Theta =tan^{-1} \frac{Im(\hat{Z_{xy}})}{Re(\hat{Z_{xy}})}
+
+for a half-space,
+
+.. math::
+    \Theta = tan^{-1} \frac{Im({Z_{xy}})}{Re({Z_{xy}})} 
+    = tan^{-1} \frac{\omega \mu}{(1-j) \sqrt{\frac{\omega \mu \sigma}{2}}} 
+    = \frac{\pi}{4}
+
+
+If \\(\\sigma \\) **increases** at depth, then \\(\\Theta \\) **increases** before returning to 45°
+
+
+If \\(\\sigma \\) **decreases** at depth, then \\(\\Theta \\) **decreases** before returning to 45°
+
+
+Survey Design
+-------------
+
+Interpretation
+--------------
+
+Pratical Consideration
+----------------------
+
+Building some Intuition for MT problem
+--------------------------------------
+
+.. raw:: html
+  :file: ./images/movieMT_time.html
```

Comment:
This is a great visual! I liked the idea you mentioned of putting this at the top and using it to motivate the discussion. The motivation is to understand what MT data are and how they are sensitive to variations in physical properties. To do this, we need to understand plane waves, so we will break down how plane waves propagate through a layered medium.